bean-validation/3.0/_index.md

```diff
@@ -1,25 +1,26 @@
 ---
-title: "Jakarta Bean Validation 3.0 (Milestone)"
-date: 2020-06-09
+title: "Jakarta Bean Validation 3.0"
+date: 2020-07-07
 summary: "Release for Jakarta EE 9"
 ---
 Jakarta Bean Validation defines a metadata model and API for JavaBean and method validation.
 
 * [Jakarta Bean Validation 3.0 Release Record](https://projects.eclipse.org/projects/ee4j.bean-validation/releases/3.0)
   * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
-* [Jakarta Bean Validation 3.0 Specification Document](./bean-validation-specification-3.0.0-M1.pdf) (PDF)
-* [Jakarta Bean Validation 3.0 Specification Document](./bean-validation-specification-3.0.0-M1.html) (HTML)
+* [Jakarta Bean Validation 3.0 Specification Document](jakarta-bean-validation-spec-3.0.pdf) (PDF)
+* [Jakarta Bean Validation 3.0 Specification Document](jakarta-bean-validation-spec-3.0.html) (HTML)
 * [Jakarta Bean Validation 3.0 Javadoc](./apidocs)
-* [Jakarta Bean Validation 3.0 TCK](http://download.eclipse.org/ee4j/bean-validation/3.0/beanvalidation-tck-dist-3.0.0-M2.zip)([sig](),[sha](),[pub]())
-  * TCK Summary Results [Hibernate Validator 7.0.0.Alpha3](./TCK-summary.html)
+* [Jakarta Bean Validation 3.0 TCK](https://download.eclipse.org/jakartaee/bean-validation/3.0/beanvalidation-tck-dist-3.0.0.zip)
```

Comment:
Why is there a dist in the file name?

Reply:
There are no strict requirements on the name of the TCK.  Most of the original Java EE TCKs that came from Oracle followed a similar naming convention.  But, since BV and CDI came from RH, there names are slightly different.  Same thing for Batch coming from IBM.  This is not a big deal.